tests/js/index.js

```diff
@@ -73,6 +74,22 @@ exports.registerTests = function(tests) {
     }
 };
 
+exports.prepare = function(done) {
+    if (!global.isNodeProcess || global.testAdminUserInfo) {
+        done();
+    }
+
+    let helper = require('./admin-user-helper');
+    helper.createAdminUser().then(userInfo => {
+        global.testAdminUserInfo = userInfo;
```

Comment:
What if we have the tests call `createAdminUser()` directly to always get a fresh admin user?